Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -159,7 +159,34 @@ MediaTime MediaPlayerPrivateGStreamerMSE::durationMediaTime() const
 void MediaPlayerPrivateGStreamerMSE::seek(const MediaTime& time)
 {
     GST_DEBUG_OBJECT(pipeline(), "Requested seek to %s", time.toString().utf8().data());
-    doSeek(time, m_playbackRate, GST_SEEK_FLAG_FLUSH);
+
+    if (m_isSeekPending) {
+        if (m_seekTime != time) {
+            m_seekTime = time;
+            m_mediaSource->seekToTime(m_seekTime);
```

Comment:
Why do you need to call seekToTime() here...

Reply:
In order to anticipate the content download from the seek position in the MSE part.
This PR put just the seek request on pending in the GStreamer part and not in the MSE part.